.github/workflows/release.yaml

```diff
@@ -0,0 +1,24 @@
+name: release
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - uses: actions/setup-python@v5
+      with:
+        python-version: 3.x
+
+    - run: pipx run build
```

Comment:
Can we use `python -m build` directly instead of `pipx`? Also, can we please add a `twine check --strict dist/*` step?

Reply:
`build` is not a stdlib and need to be installed with `pip`, which is what `pipx run build` mean, pip install it and run it

Reply:
Well, I suppose we just need to install it first. I'll add it to the `tox.ini` commands. Thanks.

Reply:
> Well, I suppose we just need to install it first. I'll add it to the `tox.ini` commands. Thanks.

you don't need to add it...

Reply:
I certainly don't need to, but I prefer to regroup the commands and dependencies in one-place.